doc/radosgw/keystone.rst

```diff
@@ -54,15 +54,20 @@ only use implicit tenants, and the other protocol will
 never use implicit tenants.  Some older versions of ceph
 only supported implicit tenants with swift.
 
-Ocata (and later)
+Ocata (and Later)
```

Comment:
Caps in headings are something we admittedly are not consistent about.  I would tend to favor capitalization as you have it here, so long as that doesn't break links elsewhere - I don't know if link references are case-sensitive.

Reply:
Good point and something I did not even consider.

https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html says:
> Reference names are whitespace-neutral and case-insensitive.